packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/index.tsx

```diff
@@ -73,7 +73,7 @@ const ConfirmResetPassword = ({
         currentPasswordForgotToken
       );
       if (!isValid) {
-        hardNavigateToContentServer('/signin');
+        hardNavigateToContentServer('/');
```

Comment:
Do we want to preserve query params here?

Reply:
Is this change needed? If we redirect to `/signin` and they shouldn't be there, users will be redirected to `/`. (Aren't we redirecting to `/signin` in Backbone?)

I think we can pass the params for flow params, but I'm not sure if they're needed.

Reply:
You're right, we should keep this navigation as signin in case a user has their account in local storage. I've just updated the test to reflect that it will redirect before rendering the signin page if there is no account in local storage.